exec_game.cc

```diff
@@ -17,29 +17,27 @@ int main(int argc, char* argv[]) {
     sticksCount = std::atoi(argv[playerCount + 2]);
   }
   std::vector<std::string> listOfNames;
-  std::vector<int> sticksAllowed();
-  for(int i = 2; i < playerCount + 2; ++i) {
+  std::vector<int> sticksAllowed;
+  for (int i = 2; i < playerCount + 2; ++i) {
     listOfNames.emplace_back(argv[i]);
   }
-  for(int i = playerCount+3; i < argc; ++i) {
+  for (int i = playerCount+3; i < argc; ++i) {
     sticksAllowed.emplace_back(std::atoi(argv[i]));
   }
 
   std::unique_ptr<gameofsticks::Session> the_game;
-  if(sticksCount && !sticksAllowed.empty()) {
-    the_game(
-        new gameofsticks::Session(
-                gameofsticks::GameType::PLAYER_V_PLAYER,
-                listOfNames,
-                sticksNumber,
-                sticksAllowed));
-  }
-  else {
-    the_game(
-           new gameofsticks::Session(
-                 gameofsticks::GameType::PLAYER_V_PLAYER,
-                 listOfNames));
+  if (sticksCount && !sticksAllowed.empty()) {
+    the_game = std::unique_ptr<gameofsticks::Session>(
```

Comment:
when I pulled the code you pushed to master and tried to build it, it didn't actually compile due to syntactical issues with how the game session was being instantiated with a unique_ptr. I was confused because I thought we got it to work during our session? this syntax fix worked for me when I tried to recompile; feel free to check out bogu/master and compiling to confirm.